src/Pages/Main/MainSlide.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import Carousel from 'react-material-ui-carousel';
+import SlideItem from './components/SlideItem';
+
+export default function MainSlide({ slideData }) {
+  return (
+    <div>
+      <WrapCarousel>
+        <Carousel
+          className="carousel"
+          stopAutoPlayOnHover={false}
+          interval={8000}
+          animation={'slide'}
+          fullHeightHover={true}
+          timeout={(400, 150, 150)}
+          navButtonsProps={{
+            style: {
+              backgroundColor: 'pink',
+              color: 'black',
+            },
+          }}
+          indicatorIconButtonProps={{
+            style: {
+              marginTop: '-70px',
+              marginRight: '10px',
+              color: 'white', // 3
+            },
+          }}
+          activeIndicatorIconButtonProps={{
+            style: {
+              color: '#EB91E3',
+            },
+          }}
+        >
```

Comment:
```suggestion
const settings = {
  className: carousel,
  stopAutoPlayOnHover: false,
  interval: 8000,
  animation: 'slide',
  fullHeightHover: true,
  timeout: (400, 150, 150),
  navButtonsProps: {
    style: {
      backgroundColor: 'pink',
      color: 'black',
    },
  }}
  indicatorIconButtonProps: {
    style: {
      marginTop: '-70px',
      marginRight: '10px',
      color: 'white', // 3
    },
  }}
  activeIndicatorIconButtonProps: {
    style: {
      color: '#EB91E3',
    },
  }}
...

        <Carousel {...settings} >
```
슬라이더 관련 세팅들은 컴포넌트 밖에 settings라는 객체에 분리해두고 스프레드 연산자로 전달해주면 깔끔할 거 같습니다

Reply:
스프레드 연산자로 뺴주고 관리하니깐 정말 깔끔하고 보기좋아졌어요!